src/gui/editors/SongEditor.cpp

```diff
@@ -824,15 +828,16 @@ void SongEditor::updatePosition( const TimePos & t )
 		m_positionLine->hide();
 	}
 
-	m_positionLine->setFixedHeight( height() );
+	m_positionLine->setFixedHeight( calcTotalHeightOfTracks() );
+
 }
 
 
 
 
 void SongEditor::updatePositionLine()
 {
-	m_positionLine->setFixedHeight( height() );
+	m_positionLine->setFixedHeight( calcTotalHeightOfTracks() );
```

Comment:
You *could* do the height calculation here directly I suppose, since the position line is only found in Song Editor. But there are some benefits of having a general function I suppose, if the Pattern Editor gets a position line in the future...

Reply:
PS when you change a line, take the opportunity to update the coding style - in this case remove  the whitespace.